include/swift/SIL/SILBasicBlock.h

```diff
@@ -557,6 +557,9 @@ public SwiftObjectHeader {
   /// Pretty-print the SILBasicBlock.
   void dump() const;
 
+  /// Pretty-print the SILBasicBlock with Debug Info.
+  void dump(bool DebugInfo) const;
```

Comment:
What's the difference between `dump()` and `dump(false)`?
Or did you add a new function because the debuggers don't deal gracefully with default arguments?

Reply:
`dump()` and `dump(false)` will behave the same, I added the new function to make default arguments not be an issue yeah